documentation/src/docs/asciidoc/release-notes/release-notes-5.2.0-M1.adoc

```diff
@@ -39,7 +39,8 @@ on GitHub.
 
 ==== New Features and Improvements
 
-* ❓
+* `@MethodSource` now supports external classes factory methods referenced by fully
```

Comment:
Please change to:

`@MethodSource` now supports `static` factory methods declared in external classes referenced by _fully qualified method name_.